lib/tyk/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+
+services:
+  tyk:
+    image: tykio/tyk-gateway:v2.9.3.1
+    #networks:
+    #  - ${DOCKER_NET}
+    ports:
+      - "8080:8080"
+    deploy:
+      placement:
+        constraints:
+          - node.role == manager
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+        max_attempts: 3
+        window: 120s
+    volumes:
```

Comment:
This strategy of volume configuration is not recommended, instead, consider creating a `tyk-gateway` volume with `middleware`, `apps`, and `policies` subdirectories. this way you can have 1 volume mount to `/opt/tyk-gateway`